src/PrestaShopBundle/Form/Admin/Improve/International/Geolocation/GeolocationOptionsFormDataProvider.php

```diff
@@ -72,7 +72,15 @@ public function __construct(
      */
     public function getData()
     {
-        return $this->dataConfiguration->getConfiguration();
+        $configuration = $this->dataConfiguration->getConfiguration();
+
+        if (!empty($configuration['geolocation_countries'])) {
+            $configuration['geolocation_countries'] = explode(';', $configuration['geolocation_countries']);
+        } else {
+            $configuration['geolocation_countries'] = [];
+        }
```

Comment:
Just a proposal, nothing mandatory here :)
```suggestion
        $configuration['geolocation_countries'] = explode(';', $configuration['geolocation_countries'] ?? '');
```

Reply:
I think the explode function always returns a value, it will never be null and `??` is like doing an `isset()`

also we want it to be an empty array if there is no country selected